examples/fabric-admin/commands/interactive/InteractiveCommands.h

```diff
@@ -55,14 +55,26 @@ class InteractiveStartCommand : public InteractiveCommand
     InteractiveStartCommand(Commands * commandsHandler, CredentialIssuerCommands * credsIssuerConfig) :
         InteractiveCommand("start", commandsHandler, "Start an interactive shell that can then run other commands.",
                            credsIssuerConfig)
-    {}
+    {
+#if defined(PW_RPC_ENABLED)
+        AddArgument("fabric-bridge-server-port", 0, UINT16_MAX, &mFabricBridgeServerPort,
+                    "The fabric-bridge RPC port number to connect to (default: 33002).");
+        AddArgument("local-server-port", 0, UINT16_MAX, &mLocalServerPort,
```

Comment:
Use constant instead of hard coded number 
constexpr uint16_t kFabricBridgeServerPort = 33002;
constexpr uint16_t kFabricLocalServerPort = 33001;

Reply:
Done